rust/operator-binary/src/airflow_db_controller.rs

```diff
@@ -102,6 +102,8 @@ pub enum Error {
         source: crate::product_logging::Error,
         cm_name: String,
     },
+    #[snafu(display("unable to find SecretClass with airflow credentials"))]
+    SecretNotFound,
```

Comment:
Can we mention the credential secrets name and something like "spec.credentialsSecret" in the error?

Reply:
Not sure, there is no secret and therefore no name I guess. Could mention the name of the secret in the docs.

Reply:
The credentialsSecret is a String in the CRD and is there. It just does not exist in the API server.